charts/mysql/remedy.md

```diff
@@ -0,0 +1,88 @@
+## TODOs on MySQL Critical Alerts
+
+### Database Alerts
+
+- #### MySQLInstanceDown
+  - if one instance is down, Restart that instance(pod)
```

Comment:
do we add things elaborately?
`describe the MySQL cr and check the reason and try restarting the pods`